src/libs/getClickedElementLocation/index.native.js

```diff
@@ -1,12 +1,13 @@
 /**
  * We don't need to get the position of the element on native platforms because the popover will be bottom mounted
  *
+ * @param {Object} nativeEvent
  * @returns {Object}
  */
-function getClickedElementLocation() {
+function getClickedElementLocation(nativeEvent) {
     return {
-        bottom: 0,
-        left: 0,
+        bottom: nativeEvent.nativeEvent.pageY - nativeEvent.nativeEvent.locationY,
```

Comment:
`nativeEvent.nativeEvent` looks confusing.

Reply:
Agree. Based on previous handling of this, I have changed the code that calls this method to pass the nested `nativeEvent` property instead of the whole object.